test/MessageTest.cpp

```diff
@@ -70,6 +70,42 @@ TEST(MessageTest, GetParams) {
     EXPECT_EQ(CMessage("CMD p1 :p2 p3").GetParams(-1, 10), "");
 }
 
+TEST(MessageTest, GetParamsSplit) {
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(0), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(1), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(-1), ContainerEq(VCString()));
+
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(0, 0), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(1, 0), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(-1, 0), ContainerEq(VCString()));
+
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(0, 1), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(1, 1), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(-1, 1), ContainerEq(VCString()));
+
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(0, 10), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(1, 10), ContainerEq(VCString()));
+    EXPECT_THAT(CMessage("CMD").GetParamsSplit(-1, 10), ContainerEq(VCString()));
+
+    EXPECT_THAT(CMessage("CMD p1 :p2 p3").GetParamsSplit(0), ContainerEq(VCString({"p1", "p2 p3"})));
```

Comment:
`ElementsAre("p1", "p2 p2")` would be simpler; and the same for the rest of of this test